### PR TITLE
[#hotfix-IR-328] review fixes

### PR DESCRIPTION
**Pull request by Ravi2714**

### CODE REVIEW

File: apps/retrospective/services/sprint_sync.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```
Comment (himanshu-garg-165) [inline apps/retrospective/services/sprint_sync.go:11]: Is it really required?

File: apps/tasktracker/providers.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ type Connection interface {
 	GetSprint(sprintID string) *serializers.Sprint
 	GetSprintTaskList(sprint serializers.Sprint) []serializers.Task
 	ValidateConfig() error
+	SanitizeTimeLogs([]timeTrackerSerializers.TimeLog) []timeTrackerSerializers.TimeLog
```
Comment (himanshu-garg-165) [inline apps/tasktracker/providers.go]: Task Tracker should not have any knowledge of the time provider's serializers.

File: apps/retrospective/services/sprint_sync.go [hunk 3]
```diff
@@ -380,6 +380,48 @@ func (service SprintService) GetTimeTrackerData(sprint retroModels.Sprint, taskP
...
+		return nil, nil, err
+	}
+	timeLogs, err = tasktracker.SanitizeTimeLogs(taskProviderConfig, timeLogs)
+	if err != nil {
+		utils.LogToSentry(err)
+		service.SetSyncFailed(sprint.ID)
+		return nil, nil, err
+	}
+
+	timeLogsKeymap := map[string]timeTrackerSerializers.TimeLog{}
```
Comment (himanshu-garg-165) [inline apps/retrospective/services/sprint_sync.go]: This can be optimized further and please add more comments!!

**Review by himanshu-garg-165 — CHANGES_REQUESTED** [review]

File: apps/tasktracker/providers.go (hunk 2 above)
Comment (Ravi2714) [reply apps/tasktracker/providers.go]: changed

File: apps/retrospective/services/sprint_sync.go (hunk 3 above)
Comment (Ravi2714) [reply apps/retrospective/services/sprint_sync.go]: done

**Review by himanshu-garg-165 — APPROVED** [review]

**Review by himanshu-garg-165 — APPROVED** [review]

**Pull request merged**